rocky/rocky/views/ooi_mute.py

```diff
@@ -24,3 +29,22 @@ def get_context_data(self, **kwargs):
         context["ooi_type"] = MutedFinding.get_object_type()
 
         return context
+
+
+class MuteFindingsBulkView(SingleOOIMixin):
+    ooi_class = MutedFinding
+
+    def post(self, request, *args, **kwargs):
+        selected_findings = request.POST.getlist("finding", None)
+        reason = request.POST.get("reason", None)
```

Comment:
I have thought about this and for this view I have chosen to not use class based forms because of the way it is implemented as having checkboxes in a Listview where the number of checkboxes is dependent on the number of Findings.